docker/compose_files/docker-compose.yml

```diff
@@ -133,6 +133,22 @@ services:
       - service
       - worker
 
+  shared-filesystem-worker:
```

Comment:
It's unfortunate that this has to be a completely separate thing...

Reply:
True, but:
1. There's not really a good way to pass bool arguments to python programs that are fired from docker-compose (with other arguments we can pass env variables but for bool arguments we'd have to roll our own bool argument parsing logic that would be different from python default and that would be bad too)
2. Shared fs workers require mounting the bundle store as a volume as well, so there's some difference in the container setup too. We could make all workers mount that volume, but I think this way the differences are more concrete between the two setups.

Besides, this way it's obvious from the container name whether the worker is shared fs or not.

Reply:
Yeah, that's fine.